addons/web/static/src/js/widgets/data_export.js

```diff
@@ -14,7 +14,7 @@ var _t = core._t;
 var DataExport = Dialog.extend({
     template: 'ExportDialog',
     events: {
-        'click .o_expand_parent': function(e) {
+        'click .o_expand': function(e) {
```

Comment:
No. Fix the toggling of icons instead, there is no reason to create a redundant-but-slightly-different class because the icon toggling doesn't work OOTB.

Reply:
We create new class `.o_expand` because if we use class `.o_expand_parent` for both (icon and parent row) then style(css) of `.o_expand_parent` will be applied to parent row too.